### PR TITLE
Files and executable folder changed in bootstrap.sh

### PR DESCRIPTION
**Pull request by AxelCouturierCIBM**

### CODE REVIEW

**Comment by michaelkain** [comment]
Dear @AxelCouturierCIBM thank you a lot for your PR!
I have assigned Anthony to it, who is the original developer of the bootstrap.sh.
Please be aware, that today we publish already versioned docker images on GitHub,
so there might be no need anymore to compile within the bootstrap.sh, see here:
https://github.com/orgs/fli-iam/packages?repo_name=shanoir-ng
@a-ba maybe it might be interesting to share some aspects of shanoir-deploy?
With kind regards, Michael

File: bootstrap.sh [hunk 1]
```diff
@@ -93,7 +93,7 @@ if [ -n "$build" ] ; then
 	docker build -t "$DEV_IMG" --target=jdk docker-compose
 
 	# 2. run the maven build
-	mkdir -p /tmp/home
+	mkdir -p /src/tmp/home
```
Comment (a-ba) [inline bootstrap.sh:96]: I would rather just put `mkdir -p tmp/home` here. The /src directory is inside the container (and mapped to the current directory with `-v "$PWD:/src"`), there is no point to create it outside the container (neither to force the user to put his sources into /src)

File: README.md [hunk 2]
```diff
@@ -178,7 +178,7 @@ The **bootstrap.sh** script automates the build and the deployment of shanoir.
 volumes configured in the docker-compose.yml). It is not recommended to use on a
 production host.
 
-To deploy shanoir from scratch on a development machine you can just launch the
+To deploy shanoir from scratch on a development machine you can just put the files in /src/ folder, launch the
```
Comment (a-ba) [inline README.md:181]: disagree (see below)

**Review by a-ba — CHANGES_REQUESTED** [review]

**Comment by michaelkain** [comment]
Hi @a-ba, I hope you are doing fine. Could you please give us an update? Thank you, Michael

**Comment by a-ba** [comment]
I am still waiting for the requested changes

**Pull request closed without merging**